### PR TITLE
Modify ReadSourceData function to trigger pipeline directly

### PR DESCRIPTION
**Pull request by DanPaseltiner**
This PR modifies the read source data Azure Function to trigger the ADF ingestion pipeline directly for every message in a source file.

### CODE REVIEW

**Review by nickclyde — APPROVED** [review]
This looks great, good work on this Dan! Hopefully this works on first deploy 🤞

**Review by BradySkylight — APPROVED** [review]
LGTM

File: quick-start.ps1 [hunk 1]
```diff
@@ -113,7 +113,7 @@ $APP_REG_NAME = "github"
 $CLIENT_ID = (az ad app create --display-name $APP_REG_NAME --query appId --output tsv)
 
 # Create service principal and grant access to subscription
-az ad sp create-for-rbac --scopes /subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP_NAME --role owner --name $APP_REG_NAME
+az ad sp create-for-rbac --scopes /subscriptions/$SUBSCRIPTION_ID --role owner --name $APP_REG_NAME
```
Comment (BradySkylight) [inline quick-start.ps1:116]: Good catch here!

File: quick-start.sh [hunk 2]
```diff
@@ -105,7 +105,7 @@ APP_REG_NAME=github
 CLIENT_ID=$(az ad app create --display-name $APP_REG_NAME --query appId --output tsv)
 
 # Create service principal and grant access to subscription
-az ad sp create-for-rbac --scopes /subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP_NAME --role owner --name $APP_REG_NAME
+az ad sp create-for-rbac --scopes /subscriptions/$SUBSCRIPTION_ID --role owner --name $APP_REG_NAME
```
Comment (BradySkylight) [inline quick-start.sh:108]: Good catch here!

**Review by nickclyde — APPROVED** [review]

**Pull request merged**